Vagrantfile

```diff
@@ -1,22 +1,13 @@
-Vagrant::Config.run do |config|
-  config.vm.box = "precise32"
-  config.vm.box_url = "http://files.vagrantup.com/precise32.box"
+VAGRANTFILE_API_VERSION = "2"
 
-  config.vm.forward_port 8080, 8080
-
-  config.vm.provision :chef_solo do |chef|
-    chef.cookbooks_path = [".cookbooks", "cookbooks"]
-    
-    chef.add_recipe "skylines"
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 512
+    v.cpus = 1
+    v.gui = true
```

Comment:
afaik it's using more resources, so I'm not a big fan of it if it's not strictly needed